gobblin-cluster/src/main/java/org/apache/gobblin/cluster/HelixRetriggeringJobCallable.java

```diff
@@ -263,7 +275,7 @@ private void runJobExecutionLauncher() throws JobException {
       jobLock.lock();
 
       try {
-        if (planningJobIdFromStore.isPresent() && !canRun(planningJobIdFromStore.get(), planningJobManager)) {
+        if (planningJobIdFromStore.isPresent() && !canRun(planningJobIdFromStore.get(), planningJobHelixManager)) {
```

Comment:
not pressing, but equivalent:
```
if (planningJobIdFromStore.map(id -> !canRun(id, planningJobHelixManager)).getOrElse(false)) {
```

Reply:
Actually I tried that, here and at other place. But because the map Function does not expect to throw an exception, it does not end up as clean as this.

Reply:
aah, darn, yes, frustrating...